bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveWorkItem.java

```diff
@@ -61,14 +61,19 @@ public String toString() {
 
     @Override
     public boolean concurrentWith(WorkItem other) {
-        if (!(other instanceof ArchiveWorkItem)) {
-            return true;
+        if (!(other instanceof ArchiveWorkItem otherArchiveItem)) {
+            if (!(other instanceof LabelsUpdaterWorkItem otherLabelsUpdaterItem)) {
```

Comment:
Pattern-matching `instanceof` was finalized in JDK 16. Has Skara already bumped its minimum JDK to 16?

Reply:
We are running the Skara bots on JDK 16, but officially we are still listing JDK 14 as minimum. I don't think it's worth bumping the minimum for this.

Reply:
In that case, you will need to use explicit casts here rather than the pattern-matching `instanceof`.

Reply:
Doh, now I understand what you mean. I must have my intellij configured incorrectly and let it just rewrite this to pattern matching style without thinking about it. Then I was out sick for a week and forgot about it.

Reply:
I changed my settings and tried compiling with JDK 14 and discovered that we have already have several JDK 15 and 16 dependencies that have crept in. Given that, I think I will just bump the required version to 16 (as a separate change). Skara has always been close to the bleeding edge anyway.